config/detekt.yml

```diff
@@ -168,4 +168,8 @@ style:
   UselessCallOnNotNull:
     active: true
   UtilityClassWithPublicConstructor:
+    active: true
+
+TbdRuleset:
```

Comment:
Is this the name other people would use if we "publish" this ruleset in the marketplace?

Reply:
yep! if we were to publish this, we can publish the ruleset as well as the rules under it. 

definitely open to changing the name of the ruleset. i'm bad at naming things. 😅 

we'd make changes to [this doc](https://github.com/detekt/detekt/blob/main/website/src/data/marketplace.js) in order to publish our ruleset to the detekt's marketplace